app/src/project/components/giveTime/giveTime.actions.js

```diff
@@ -1,38 +1,20 @@
-import { getGraphQL } from '../../../common/common.actions'
+import { PostRequest } from '../../../common/common.actions'
 import * as constants from './giveTime.actionTypes'
 
-export function giveTime ({ userToken, amount, projectRowId }) {
+export function giveTime ({ userToken, userId, amount, projectId }) {
```

Comment:
We cannot let the client sent a userId, this value should be set server-side by the authentication system